packages/lazy-fn-vite-plugin/src/constants.ts

```diff
@@ -1,3 +1,3 @@
 export const TUONO_DYNAMIC_FN_ID = 'dynamic'
-export const TUONO_LAZY_FN_ID = 'lazyLoadComponent'
+export const TUONO_LAZY_FN_ID = 'tuono__internal__lazyLoadComponent'
```

Comment:
```suggestion
export const TUONO_LAZY_FN_ID = '__tuono__internal__lazyLoadComponent'
```

I would consider using a `__` prefix to avoid suggestion when typing  `tuono` since this type is exported,

I have a remembrance that we have discussed this, but probably I have just imagined it 😅